From 2aba472f96acd23a0a3cb92a20a60301a6fc23fb Mon Sep 17 00:00:00 2001
Message-Id: <2aba472f96acd23a0a3cb92a20a60301a6fc23fb.1421395644.git.chang-joon.lee@intel.com>
In-Reply-To: <87b3fd6ee09eca535f11edc6180f37fb5c67ac4d.1421395644.git.chang-joon.lee@intel.com>
References: <87b3fd6ee09eca535f11edc6180f37fb5c67ac4d.1421395644.git.chang-joon.lee@intel.com>
From: Dave Gordon <david.s.gordon@intel.com>
Date: Mon, 20 Oct 2014 14:21:15 +0100
Subject: [PATCH 7/7] FOR_UPSTREAM [VPG]: enable full PPGTT by default on GEN8

Remove the temporary code that disabled full PPGTT on GEN8

Issue: GMINL-4336
Change-Id: I4d3a0f0a2d55866e4ec45bcd7e7ea20502a8a00f
Signed-off-by: Dave Gordon <david.s.gordon@intel.com>
Signed-off-by: Michel Thierry <michel.thierry@intel.com>
---
 drivers/gpu/drm/i915/i915_gem_gtt.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_gtt.c b/drivers/gpu/drm/i915/i915_gem_gtt.c
index 2ed9d01..fc8bfc5 100644
--- a/drivers/gpu/drm/i915/i915_gem_gtt.c
+++ b/drivers/gpu/drm/i915/i915_gem_gtt.c
@@ -40,8 +40,6 @@ static int sanitize_enable_ppgtt(struct drm_device *dev, int enable_ppgtt)
 
 	has_aliasing_ppgtt = INTEL_INFO(dev)->gen >= 6;
 	has_full_ppgtt = INTEL_INFO(dev)->gen >= 7;
-	if (IS_GEN8(dev))
-		has_full_ppgtt = false; /* XXX why? */
 
 	if (enable_ppgtt == 0 || !has_aliasing_ppgtt)
 		return 0;
-- 
1.7.9.5

